src/DonorProfiles/resources/js/app/tabs/donation-history/content.js

```diff
@@ -38,6 +38,9 @@ const Content = () => {
 				<p style={ { color: '#6b6b6b' } }>
 					{ error }
 				</p>
+				<p style={ { color: '#6b6b6b' } }>
```

Comment:
@henryholtgeerts I hardcoded this additional info, hope this approach is ok? I was also thinking to wrap everything in one `Error` component, but I'm not sure about that. Let me know what you think.

Reply:
I think this is fine, since it's the only place an error message occurs right now. 👍